tests/unit/stream_alert_shared/test_rule_table.py

```diff
@@ -0,0 +1,281 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from datetime import datetime
+import os
+from StringIO import StringIO
+
+from mock import Mock, patch
+from moto import mock_dynamodb2
+from nose.tools import assert_equal
+
+from stream_alert.shared import rule as rule_module, rule_table
+from stream_alert_cli.helpers import setup_mock_rules_table
+
+_RULES_TABLE = 'PREFIX_streamalert_rules'
+
+
+class TestRuleTable(object):
+    """Tests for shared/rule_table.py"""
+    # pylint: disable=protected-access
+
+    @patch.dict(os.environ, {'AWS_DEFAULT_REGION': 'us-east-1'})
+    def setup(self):
+        """Rule Table - Create mock table and rules"""
+        # pylint: disable=attribute-defined-outside-init
+        self.dynamo_mock = mock_dynamodb2()
+        self.dynamo_mock.start()
+        setup_mock_rules_table(_RULES_TABLE)
+        self.rule_table = rule_table.RuleTable(_RULES_TABLE)
+
+    def teardown(self):
+        """Rule Table - Destroy previously created rules"""
+        rule_module.Rule._rules.clear()
+        self.dynamo_mock.stop()
+
+    @classmethod
+    def _create_local_rules(cls, count=5):
+        """Helper to create N fake local rules"""
+        for i in range(count):
+            cls._create_local_rule_with_name('fake_rule_{:02d}'.format(i))
+
+    @classmethod
+    def _create_local_rule_with_name(cls, name):
```

Comment:
Could this be `@staticmethod` instead of `@classmethod`?

Reply:
it could probably yeah - will make the change